linera-views/src/memory.rs

```diff
@@ -255,6 +396,10 @@ pub enum MemoryStoreError {
     #[error("The value is too large for the MemoryStore")]
     TooLargeValue,
 
+    /// The namespace is not existent
+    #[error("The namespace is not existent")]
```

Comment:
"not existent" -> "does not exist" or "is absent" (or "is non-existent" but the tone is not right)

Reply:
Corrected.